### PR TITLE
Simple Update in AxesViewer to accept line thickness arg

### PR DESCRIPTION
**Pull request by mrlooi**

### CODE REVIEW

**Review by sebavan — APPROVED** [review]
missing the comments in the @param list but looks good to me

**Review by RaananW — APPROVED** [review]

**Comment by RaananW** [comment]
it does feel like the constructor is geeing bigger than we would want it to be, but there is currently no way out of this.

**Comment by mrlooi** [comment]
> @mrlooi - there are docs issues and linting issues. care to check those? Thanks :-)

Oh I edited this directly via github, how do I check the linting stuff? Might need some help here

**Comment by RaananW** [comment]
It is just missing the documentation for the parameter lineThickness

File: src/Debug/axesViewer.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class AxesViewer {
      * @param yAxis defines the node hierarchy used to render the y-axis
      * @param zAxis defines the node hierarchy used to render the z-axis
      */
```
Comment (RaananW) [inline src/Debug/axesViewer.ts:54]:
```suggestion
     * @param lineThickness The line thickness to use when creating the arrow. defaults to 1.
     */
```

Comment (RaananW) [reply src/Debug/axesViewer.ts:54]: @mrlooi - You can merge this, unless you want to write your own documentation line :-)

**Pull request merged**